Integrations/integration-PagerDuty.yml

```diff
@@ -0,0 +1,475 @@
+description: Alert and notify users using PagerDuty
+configuration:
+- display: API Key
+  name: APIKey
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Service Key (for triggering events only)
+  name: ServiceKey
+  defaultvalue: ""
+  type: 0
+  required: false
+- display: Fetch incidents
+  name: isFetch
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Incident type
+  name: incidentType
+  defaultvalue: ""
+  type: 13
+  required: false
+script:
+  script: |
+    ''' IMPORTS '''
+    import os
+    import json
+    import requests
+    import datetime
+
+    ''' GLOBAL VARS '''
+    USE_SSL = True
```

Comment:
@ronykoz do we **always** use SSL? Usually, it is a configurable parameter

Reply:
Yes, this is from the their documentation.
```To prevent accidentally sending sensitive information over the Internet, API v2 requests require SSL. Connections made over HTTP will be refused.```

Reply:
@ronykoz ok, please add a comment in the code that clarifies that they enforce SSL usage. (to prevent future unnecessary changes)

Reply:
@ronykoz please change to:
`# Pagerduty API works only with secured communication.`

Reply:
Done